mkosi

```diff
@@ -937,6 +937,10 @@ def reset_machine_id(args, workspace, run_build_script):
         machine_id = os.path.join(workspace, 'root', 'etc/machine-id')
         os.unlink(machine_id)
         open(machine_id, "w+b").close()
+        dbus_machine_id = os.path.join(workspace, 'root', 'var/lib/dbus/machine-id')
+        if os.path.exists(dbus_machine_id):
+            os.unlink(dbus_machine_id)
+            os.symlink('../../../etc/machine-id', dbus_machine_id)
```

Comment:
hmm, smells like TOCTOU... I think it would be much nicer if we'd just unlink the thing, and if that results in a file not found exception we should just eat that up...